docs/Overrides/NginxProxyManager.md

```diff
@@ -0,0 +1,98 @@
+# Nginx Proxy Manager
+[Nginx Proxy Manager](https://[https://nginxproxymanager.com/](https://nginxproxymanager.com/))  is  Docker container for managing Nginx proxy hosts and SSL Certificates with a simple, powerful interface.
+ 
+# Features of Nginx Proxy Manager
+ 
+### Get Connected
+Expose web services on your network · Free SSL with Let's Encrypt · Designed with security in mind · Perfect for home networks
+
+### Proxy Hosts
+Expose your private network Web services and get connected anywhere.
+
+### Beautiful UI
+Based on Tabler, the interface is a pleasure to use. Configuring a server has never been so fun.
+
+### Free SSL
+Built in Let’s Encrypt support allows you to secure your Web services at no cost to you. The certificates even renew themselves!
+
+### Multiple Users
+Configure other users to either view or manage their own hosts. Full access permissions are available.
```

Comment:
I don't think it's necessary to copy/paste info like this. All of that info can be found at their website or git repo. Also those sources would be kept up to date without any changes required on our part.

Reply:
Ok. Deleted this. Works for me. lol